### PR TITLE
Status endpoint and AddPositionToCatalogue support

### PR DESCRIPTION
**Pull request by cbrit**

### CODE REVIEW

File: steward/src/cellars.rs
Comment (EricBolten) [inline steward/src/cellars.rs:24]: I suppose we can leave these two here with empty lists in case we need to do it again pre-v7, but if these lists are empty we should be able to delete it from the API entirely.

File: proto/steward.proto
Comment (EricBolten) [inline proto/steward.proto]: In retrospect it's kinda weird that the `Status` call is in the `ContractCall` service but I don't know if it's worth changing.

**Review by EricBolten — APPROVED** [review]
Small non-blocking comments, approving the PR but we'll want to wait for governance approval before merging.

**Review by josephterrigno — APPROVED** [review]

**Review by EricBolten — APPROVED** [review]
LGTM

**Pull request merged**
